scene/resources/sprite_frames.cpp

```diff
@@ -224,6 +224,23 @@ void SpriteFrames::_set_animations(const Array &p_animations) {
 	}
 }
 
+void SpriteFrames::get_argument_options(const StringName &p_function, int p_idx, List<String> *r_options) const {
+	String pf = p_function;
+	if (p_idx == 0) {
+		if (pf == "has_animation" || pf == "remove_animation" || pf == "rename_animation"
```

Comment:
I think you need to move the `||` to the end of the previous row, that works elsewhere

Reply:
Oh that's true. That's it. It's not in my style, hence I forgot to.